src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelRegulationLegend.tsx

```diff
@@ -42,6 +43,22 @@ const TrackPanelRegulationLegend = (props: { disabled: boolean }) => {
   const dispatch = useDispatch();
   const { trackDrawerOpened, reportTrackPanelSectionToggled } =
     useGenomeBrowserAnalytics();
+  const { data: regulationStats } = useGetSpeciesStatisticsQuery(
```

Comment:
It is not just the regulation stats that you will be getting here. You will get the whole species stats object, which has `regulation_stats` as a key inside of it.

By treating the whole species stats object as regulation stats, you are currently getting the following in the sidebar.

![image](https://github.com/Ensembl/ensembl-client/assets/6834224/915b3da6-dc28-4214-a95d-2f3fe13a48ad)

Reply:
Oops! sorry my bad! I think my browser wasn't responding to this for some reason. Will update it